x-pack/plugins/observability_solution/synthetics/common/saved_objects/private_locations.ts

```diff
@@ -5,5 +5,7 @@
  * 2.0.
  */
 
-export const privateLocationsSavedObjectId = 'synthetics-privates-locations-singleton';
-export const privateLocationsSavedObjectName = 'synthetics-privates-locations';
+export const legacyPrivateLocationsSavedObjectId = 'synthetics-privates-locations-singleton';
+export const legacyPrivateLocationsSavedObjectName = 'synthetics-privates-locations';
+
+export const privateLocationsSavedObjectName = 'synthetics-privates-locations-multiple';
```

Comment:
Do we want to use this change as an opportunity to match the language in description? "private" vs "privates"
```suggestion
export const privateLocationsSavedObjectName = 'synthetics-private-locations-multiple';
```